surefire-its/src/test/resources/junit-platform-engine-jupiter/pom.xml

```diff
@@ -57,10 +58,20 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.7.0</version>
```

Comment:
Why not 3.8.0?

Reply:
not sure why, i grabbed the one. 
I will upgrade it to `3.8.0`.